docs/index.html

```diff
@@ -9,12 +9,13 @@
     <title>EasySpeech - Cross browser Speech Synthesis with easy API</title>
 </head>
 <body>
-<h1 class="text-center">EasySpeech</h1>
-<div class="w-100 text-center">
+    <div class="bg-dark pb-3 navbar-nav">
+<h1 class="text-center text-white "><a href="#" class="text-decoration-none text-white" >EasySpeech</a></h1>
```

Comment:
Please use correct indentation. When you introduce a new parent then elements should be indented one level.